cookbooks/zarr-stac-report/data-producers/index.qmd

```diff
@@ -0,0 +1,277 @@
+---
+title: "Zarr + STAC for Data Producers"
+subtitle: "Approaches to structuring your data cubes using Zarr + STAC"
+---
+
+This section is targeted at people who are trying to disseminate data cubes. It addresses questions like:
+
+ - Should you use STAC?
+ - Should you use Zarr?
+ - Where should Zarr stores go in your STAC catalog?
+ - How much metadata should you pull up out of the Zarr dataset?
+ - How to pull metadata out of Zarr?
+
+## Is your data spatial-temporal?
+
+Since this is explicitly a geospatial guide, your data is probably spatial-temporal, but just in case it is not - there is a really interesting discussion going on about a general catalog that would cover a larger set of things (read more: [FROST][1]). 
+
+If you are looking for something that exists now:
```

Comment:
something that does what?

Reply:
I moved things around a bit to make it clear that this section is about whether you should use STAC